config/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
let's put webpack.config.js in the root folder... not enough configs for a `config` folder.